src/index.css

```diff
@@ -1,3 +1,66 @@
+* {
+  margin: 0%;
+  padding: 0%;
+}
+
```

Comment:
- [ ] OPTIONAL
The calculator on the calculator page is ** not properly styled**. The display of the calculator should match the width of the entire calculator, whenever there is a breakpoint in the page. Kindly ensure to improve upon aspect.


**Your Calculator page:**
![image](https://user-images.githubusercontent.com/12745474/141202169-5661c36f-7d07-4e4c-9e1e-ebb6c15d626c.png)


![image](https://user-images.githubusercontent.com/12745474/141202301-208c7246-971b-4112-95b8-c4944389d049.png)